core/templates/dev/head/pages/splash-page/splash-page.directive.html

```diff
@@ -0,0 +1,165 @@
+<div>
```

Comment:
Do you need this external div?

Reply:
This was present in the original file so I decided not to tamper with it lest it should break the UI.